src/aleph/vm/sevclient.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+from aleph.vm.utils import run_in_subprocess
+
+
+class SevClient:
+    def __init__(self, sev_dir: Path):
+        self.sev_dir = sev_dir
+        self.certificates_dir = sev_dir / "platform"
+        self.certificates_dir.mkdir(exist_ok=True, parents=True)
+        self.certificates_archive = self.certificates_dir / "certs_export.cert"
+
+    async def sevctl_cmd(self, *args) -> bytes:
+        result = await run_in_subprocess(
+            ["sevctl", *args],
+            check=True,
+        )
+
+        return result
```

Comment:
Method 'sevctl_cmd' may be 'static' . It would be better to make this method a separate function since it does not depend on the class.

Reply:
Yes, some methods can be static, but I prefer to have it in the same class, specially if we will add the `SEV_CTL_PATH` in the future.

Reply:
My IDE will keep complaining but I mark this as resolved ;-)